<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Show_hours</class>
 <widget class="QDialog" name="Show_hours">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>276</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Enter Hours</string>
  </property>
  <widget class="QLabel" name="course_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Course Hours</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="course_lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="other_label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>40</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Other Hours</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="other_lineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="update_btn">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cancel_btn">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Hour_comboBox">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>70</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Wage_lineEdit">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="wage_label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wages</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
